packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -849,7 +849,7 @@ module.exports.external = [
       try {
         return getDateTime(dateTimeString);
       } catch (err) {
-        throw `helper "formatAsDateTime" : ${err}`;
+        console.log(`helper "formatAsDateTime" : ${err}`);
```

Comment:
no need to throw error. question is why the regex replacements in preprocessing step are not catching these cases 🤔